dist/README.md

```diff
@@ -1,11 +0,0 @@
-# !! WARNING !!
-
-`cf-extension.css` file in this directory is a prebuilt committed version of legacy
```

Comment:
We're dropping this warning - but leaving the file? Is there a reason for that

Reply:
We should not drop the files, but only the final sentence

> We can remove it as soon as we reach 4.0.

And update the unpkg url maybe